lib/components/flat_button_long.dart

```diff
@@ -26,7 +26,7 @@ class FlatButtonLong extends StatelessWidget {
                 width: 17,
                 height: 17,
                 child: const CircularProgressIndicator(
-                  // color: AppColors.white,
```

Comment:
<img width="456" alt="image" src="https://user-images.githubusercontent.com/8041901/193460934-41a3b218-2802-4238-9876-bdc16120c45d.png">
This was causing the indicator to not be visible